### ISSUE TITLE
Update 3.0.1 TGCD / SEGACD / NEOCD refuse to launch

### ISSUE THREAD

**Issue by Cranmoot** [spruceUI/spruceOS#190]
Hello, just letting you know... as far as my experience goes.  After update to 3.0.1 I went back to check status of all emu's on 4 a30's I'm giving as Christmas gifts.  It seems that NEOCD / SEGACD / TGCD are refusing to boot on my devices, I also tried alternate emulator on segacd still no luck.  Keep up the good work though, this is great huge step up from stock.

**Comment by ryanmsartor** [comment]
I notice it's all disc-based systems. The first question that comes to mind is - do you have the correct BIOS files for these in your BIOS folder?

SEGACD:
bios_CD_E.bin (req. for PAL games)
bios_CD_J.bin (req. for JP games)
bios_CD_U.bin (req. for US games)

PCECD: syscard3.pce (required)

NEOCD: see [here](https://github.com/libretro/neocd_libretro)

**Comment by Cranmoot** [comment]
Yes all accounted for and then some... was working before update.... not all cd systems btw I've got DC and ps1 running

**Comment by ryanmsartor** [comment]
Did you update from stock or from a previous version of spruce? The BIOS locations are different for the two

**Comment by Cranmoot** [comment]
Clean install, not sure if this will work but I'll try uploading screens

![20241115_201948](https://github.com/user-attachments/assets/1b341ec5-8614-4049-9883-359ffc56aa9b)
![20241115_201934](https://github.com/user-attachments/assets/90abcf4d-0233-4024-b1f6-8915de59d7a9)
![20241115_201915](https://github.com/user-attachments/assets/bac82d32-fb5b-43bf-a7e6-e7a137053064)

**Comment by ryanmsartor** [comment]
yeah those look all good... hmm... what file format are your roms in? And, it seems to be ALL your games for those systems?

**Comment by Cranmoot** [comment]
All are .chd.... was working for last 2 updates,  came in on 2.3.0.... excellent work btw 👏,  I've been running 15 systems and mostly positive experience

**Comment by Cranmoot** [comment]
Netplay between them all,  5 player at times is amazing

**Comment by Cranmoot** [comment]
I'll do a fresh install on one,  and copy the Roms over.  Let you know after

**Comment by Cranmoot** [comment]
TLDR: Fresh rufus unzip & transfer of the systems in question fixed the issue.

Something I've noticed with the device not directly the spruce os overlay.   I had a fresh install (rufus/unzip/romtransfer), and  I don't think I've done much either through the file gui on the device, or my comp other than to change nds (audio would get desynced and even telling drastic to restart wouldn't solve it, I changed autostate to "0" to get around) beside the point, The last thing I was trying to tweak before calling them done and wrapping them up for Christmas.  Was XMen FBNeo 6 player...   It's so close to full speed & as a last resort I changed to overclocked & it indeed played fullspeed for 2 bosses, but when I tried to start it again it locked up & crashed after a few min.  I bring this up because I've done these builds several times now because I have several devices to prepare & it seems to me that when the device freezes as described above.  It continues to write garbage to the SD.  I've had a couple builds refuse to write to disk and require a format to recover after overclocking the device.  So maybe overclock is too unstable to utilize in the interface... idk just my two cents here.  

Applause to all involved.  I've thoroughly enjoyed building all these devices for my family, and I hope they are a hit at Christmas.  How far we've come since childhood, but I didn't choose to make the investment in these devices until I heard about the work you've been doing with this project.

**Comment by Sundownersport** [comment]
Thanks! Glad to hear you fixed it.

we do not recommend overclocking in general, some a30s just don’t work right with it. Performance mode is the highest recommended.

we are aware of the read only errors that happen to some users, we are actively working on a fix for this.
